guides/common/attributes.adoc

```diff
@@ -11,24 +11,20 @@
 :ProductVersion: 6.9-beta
 :ProductVersionPrevious: 6.8
 :ProductVersionRepoTitle: Beta
```

Comment:
Isn't this also a downstream only attribute that should live in Satellite's attributes?
```console
$ rg ProductVersionRepoTitle
guides/common/attributes.adoc
13::ProductVersionRepoTitle: Beta

guides/common/attributes-satellite.adoc
74::project-client-name: Satellite Tools {ProductVersionRepoTitle}
82::ProjectNameXY: Red{nbsp}Hat Satellite{nbsp}{ProductVersionRepoTitle}
85::ProjectXY: Satellite{nbsp}{ProductVersionRepoTitle}
```

Reply:
Yes, changing.